dsgrid/config/dimension_mapping_base.py

```diff
@@ -273,6 +273,12 @@ class DimensionMappingPreRegisteredBaseModel(DSGBaseModel):
         description="Tolerance value to apply to the to_fraction column",
         default=1e-6,
     )
+    from_dimension_name: Optional[str] = Field(
```

Comment:
This is a key change. Dataset contributors specify the project base dimension here.

Reply:
The code now has a field called `project_base_dimension_name`